src/tests/controllers/TagController.test.js

```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chai from 'chai';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import status from '../../config/status';
+import db from '../../models';
+import * as Factory from '../../helpers/factory';
+
+const { expect } = chai;
+chai.should();
+
+const article = Factory.article.build();
+chai.use(chaiHttp);
+const token = 'token-string';
+describe('TAGS', () => {
+  let response = '';
+  before(async () => {
+    const findUser = await db.User.findAll({ limit: 1, logging: false });
```

Comment:
you find all from User model and you limit is 1, you can use findOne instead

Reply:
With **findAll**, it is easy to return one format of the response to the controller